.vsts-dotnet-ci.yml

```diff
@@ -63,14 +48,30 @@ phases:
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: MacOS
+    name: MacOS_x64
     buildScript: ./build.sh
     queue:
       name: Hosted macOS
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Windows_NetFx
+    name: Windows_x64_1
```

Comment:
I'm not sure `_1` and `_2` help any on the readability, does it?

Reply:
I didnt want to add these suffixes  but names for the jobs need to be unique.

Reply:
They look unique to me without the suffixes:

![image](https://user-images.githubusercontent.com/8291187/56060033-47bc2400-5d2b-11e9-9f23-875d6dcdc3ae.png)

Reply:
```Debug_3.0``` comes from custom Matrix attribute (It will be always added automatically) the name is the first part before with space.

Reply:
It seems like something better should be able to be done here.  Maybe @safern or @chcosta would have some ideas.

Reply:
If you were to merge the matrices such that you were using a single job with a larger matrix, you wouldn't hit collision issues with job names.

Also, phases are now a deprecated syntax.  Azure DevOps has moved to "jobs".  There are some actual changes associated with the naming change and I'd suggest you move to the supported syntax.  Here's our (Arcade's) documentation which is related but at least will point you in the right direction - https://github.com/dotnet/arcade/blob/master/Documentation/AzureDevOps/PhaseToJobSchemaChange.md

Reply:
@eerhardt I used a bigger custom matrix. Although it increased the number of CI legs but its just more combinations of configurations.